src/v1/Controllers/Home.php

```diff
@@ -191,6 +228,185 @@ public function homepage(Request $request, Response $response, array $args): Res
     $nb_paging_total = [];
     $nb_paging_total['paging']['total'] = $nbValTotal;
     $viewData->addData('items', $items);
+
+    $myData = [
+      // [
+      //   'header' => [
+      //     'title'     => 'Last escaladed tickets',
+      //     'subtitle'  => '',
+      //     'name'      => 'last-escaladed-tickets',
+      //   ],
+      //   'list' => [
+      //     [
+      //       'id'   => '254678',
+      //       'name' => 'signature problem',
+      //     ],
+      //   ],
+      //   'footer' => [
+      //     'enabled' => true,
+      //     'url' => '',
+      //   ],
+      //   'color' => 'blue',
+      // ],
+    ];
+
+    // TODO dynamic here, delete before this line
+    $myData = [];
+    [$cnt, $data] = $this->getNewTickets();
+    $myData[] = [
+      'header' => [
+        'icon'      => 'book open',
+        'title'     => $translator->translate('New tickets'),
+        'subtitle'  => $cnt . ' tickets',
+        'name'      => 'new-tickets',
+      ],
+      'list'   => $data,
+      'footer' => [
+        'enabled' => $cnt >= 5,
+        'url'     => '',
+      ],
+      'color'  => 'olive',
+      'url'    => $basePath . '/view/tickets',
+    ];
+
+    [$cnt, $data] = $this->getTicketsAssignedToMe();
+
+    $myData[] = [
+      'header' => [
+        'icon'      => 'book reader',
+        'title'     => 'Tickets assigned to me',
+        'subtitle'  => $cnt . ' tickets',
+        'name'      => 'my-assigned-tickets',
+      ],
+      'list'   => $data,
+      'footer' => [
+        'enabled' => $cnt >= 5,
+        'url'     => '',
+      ],
+      'color'  => 'blue',
+      'url'    => $basePath . '/view/tickets',
+    ];
+
+    [$cnt, $data] = $this->getMyTickets();
+
+    $myData[] = [
+      'header' => [
+        'icon'      => 'user circle',
+        'title'     => 'My tickets',
+        'subtitle'  => $cnt . ' tickets',
+        'name'      => 'my-tickets',
+      ],
+      'list'   => $data,
+      'footer' => [
+        'enabled' => $cnt >= 5,
+        'url'     => '',
+      ],
+      'color'  => 'green',
+      'url'    => $basePath . '/view/tickets',
+    ];
+
+    [$cnt, $data] = $this->getMyGroupTickets();
+
+    $myData[] = [
+      'header' => [
+        'icon'      => 'user friends',
+        'title'     => 'Tickets of my groups',
+        'subtitle'  => $cnt . ' tickets',
+        'name'      => 'my-groups-tickets',
+      ],
+      'list'   => $data,
+      'footer' => [
+        'enabled' => $cnt >= 5,
+        'url'     => '',
+      ],
+      'color'  => 'blue',
+    ];
```

Comment:
miss url, same for next declarations, perhaps create and use datainterface to have it with right variables